### DISCUSSION TITLE
Does setting a type via cheater syntax work currently?

### DISCUSSION THREAD

**Discussion by denismaier** [retorquere/zotero-better-bibtex#2472] [General]
I've wanted to set an item to the type "performance" which is not yet supported in Zotero. However, on expert, the original item type takes over. Is that on purpose or am I missing something?

**Comment by retorquere** [comment]
I need a debug log in order to help with export questions. Right-click the item, choose "Better BibTeX" from the popup menu, then "Send debug log", and paste the ID here.

**Reply by denismaier** [reply]
2UFMJWLV-refs-euc

**Reply by retorquere** [reply]
Try `tex.entrytype: performance` in the `extra` field.

**Reply by retorquere** [reply]
is "performance" planned as a type in Zotero? In that case I could perhaps test for the `type` line.

**Reply by retorquere** [reply]
Oh I see -- artwork with type `performance` is a supported biblatex type. artwork + type seems like a reasonable match, I'll get a fix for that, shouldn't take long.

**Reply by denismaier** [reply]
Oh, I should note that my target is better CSL json

**Reply by denismaier** [reply]
Oh, and actually I sometimes have other official types selected: sometimes it's video recording with `type: performance` since I need some of those fields.

**Reply by retorquere** [reply]
Ah, that's sort of different. BBT generates bibtex/biblatex from scratch but augments the CSL generated by Zotero itself. I'll look into it, but this may be something better handled by Zotero itself (in which case BBT would inherit it automatically). But I'll know more later today.

**Reply by retorquere** [reply]
Can you try the build at https://github.com/retorquere/zotero-better-bibtex/issues/2473

**Reply by denismaier** [reply]
I've now had a chance to update to plugin, and it seems to work! Thanks for the swift change!

**Comment by retorquere** [comment]
Continued on [#2743](https://github.com/retorquere/zotero-better-bibtex/issues/2473)
